web/app/components/user-info/user-info.jsx

```diff
@@ -29,19 +30,30 @@ class UserInfo extends Component {
 
   render(props, { comments, isLoading }) {
     const {
-      user: { name, id },
+      user: { name, id, isDefaultPicture, picture },
       onClose,
     } = props;
 
     return (
       <div className={b('user-info', props)}>
+        <img
+          className={b('user-info__avatar', {}, { default: !!isDefaultPicture })}
+          src={isDefaultPicture ? CommentAvatarSrc : picture}
+          alt=""
+        />
         <p className="user-info__title">Last comments by {name}</p>
         <p className="user-info__id">{id}</p>
 
         <LastCommentsList isLoading={isLoading} comments={comments} />
 
-        <span {...getHandleClickProps(onClose)} className="user-info__close">
-          Close
+        <span
+          {...getHandleClickProps(onClose)}
+          className="user-info__close"
+          onKeyDown={onClose}
```

Comment:
ouch. getHandleClickProps already does that for you. as well as role and tabindex too. See implementation for details